packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -50,281 +51,288 @@ export const Default = {
 };
 
 /**
- * The sd-flipcard can be displayed in several ways using the `front-variant` and `back-variant` attributes. This example shows the usage `front-variant` attribute.
+ * Use the `variant` attribute to set the color variant for front or back.
+ * - empty (default)
+ * - primary
+ * - neutral-100
+ * - gradient-light-top
+ * - gradient-light-bottom
+ * - gradient-dark-top
+ * - gradient-dark-bottom
```

Comment:
```suggestion
 * Use the `variant` attribute to set the color variant for front or back.
 * - empty (default)
 * - primary
 * - neutral-100
 * - gradient-light-top
 * - gradient-light-bottom
 * - gradient-dark-top
 * - gradient-dark-bottom
 * Users can decide which interaction triggers the flip, the card will be activated by clicking or hovering or both. Flip-direction can be horizontal or vertical. Specify in the hand-off
```

![image](https://github.com/user-attachments/assets/3f2a9cf3-820a-4324-bc3b-827e39086ef0)

Reply:
Aligned with design